pkg/generic/thriftidl_provider.go

```diff
@@ -50,6 +54,17 @@ func NewThriftFileProvider(path string, includeDirs ...string) (DescriptorProvid
 	if err != nil {
 		return nil, err
 	}
+
+	if isDynamicgoGoTagAliasDisabled {
+		dthrift.RemoveAnnotationMapper(dthrift.AnnoScopeField, "go.tag")
+	}
+	// ServiceDescriptor of dynamicgo
+	dsvc, err := dthrift.NewDescritorFromPath(context.Background(), path, includeDirs...)
```

Comment:
Are you sure we CANNOT call `dynamicgo` here base on an option? It may cost more time to update IDL for users

Reply:
@AsterDY I'll use env variable to determine whether we call dynamicgo or not.

Reply:
done

Reply:
looks like a typo in the function name `NewDescritorFromPath` (missing p 😂) @AsterDY